src/app/add-item/add-item.component.html

```diff
@@ -0,0 +1,28 @@
+<div class="container">
+    <h2 class="dialogTitle" mat-dialog-title>{{ title }}</h2>
+    <mat-dialog-content [formGroup]="dialogForm" (ngSubmit)="createActionItem(dialogForm)">
+        <mat-form-field>
+            <input matInput placeholder="Action Item Name" formControlName="name" />
+        </mat-form-field>
+        <mat-form-field>
+            <mat-label>Project</mat-label>
+            <mat-select formControlName="project">
+                <mat-option *ngFor="let project of projects" [value]="project">
+                    {{ project.name }}
+                </mat-option>
+            </mat-select>
+        </mat-form-field>
+        <mat-form-field class="datePicker">
+            <input matInput [matDatepicker]="picker" placeholder="Due date" formControlName="dueDate" />
+            <mat-datepicker-toggle matSuffix [for]="picker"></mat-datepicker-toggle>
+            <mat-datepicker #picker></mat-datepicker>
+        </mat-form-field>
+        <mat-form-field class="description">
+            <textarea matInput placeholder="Description" formControlName="description"></textarea>
+        </mat-form-field>
+    </mat-dialog-content>
+</div>
+<mat-dialog-actions class="buttons">
+    <button class="mat-raised-button close" (click)="close()">Close</button>
```

Comment:
Here you are invoking `close` function without parameters but here is the signature of this function: 
```
  close(actionItem: ActionItem) {
    this.dialogRef.close(actionItem);
  }
```
Something seems to be wrong here.